recipes/libssh2/all/conanfile.py

```diff
@@ -88,8 +96,8 @@ def package(self):
         self.copy("COPYING", dst="licenses", src=self._source_subfolder, keep_path=False)
         if os.path.exists(os.path.join(self.package_folder, "lib64")):
             # rhel installs libraries into lib64
-            os.rename(os.path.join(self.package_folder, "lib64"),
-                      os.path.join(self.package_folder, "lib"))
+            tools.rename(os.path.join(self.package_folder, "lib64"),
+                         os.path.join(self.package_folder, "lib"))
```

Comment:
Seems to be useless since upstream build files rely on GNUInstallDirs:
https://github.com/libssh2/libssh2/blob/30e9c1347e3b8baa2951db612f05e6d87fc8e2f2/CMakeLists.txt#L80
https://github.com/libssh2/libssh2/blob/libssh2-1.8.0/src/CMakeLists.txt#L367-L377